### PR TITLE
Refactor: Filtering on PlantAndGardenPlanting List

### PR DESCRIPTION
**Pull request by sagar-viradiya**
- Removed filtering on PlantAndGardenPlanting List in GardenFragment and placed inside ViewModel

### CODE REVIEW

**Comment by sagar-viradiya** [comment]
I signed it

File: app/src/main/java/com/google/samples/apps/sunflower/viewmodels/GardenPlantingListViewModel.kt [hunk 1]
```diff
@@ -16,14 +16,21 @@
...
+import com.google.samples.apps.sunflower.data.PlantAndGardenPlantings
 
 class GardenPlantingListViewModel internal constructor(
         private val gardenPlantingRepository: GardenPlantingRepository
 ) : ViewModel() {
 
     fun getGardenPlantings() = gardenPlantingRepository.getGardenPlantings()
 
-    fun getPlantAndGardenPlantings() = gardenPlantingRepository.getPlantAndGardenPlantings()
+    fun getPlantAndGardenPlantings() : LiveData<List<PlantAndGardenPlantings>> {
```
Comment (tiembo) [inline app/src/main/java/com/google/samples/apps/sunflower/viewmodels/GardenPlantingListViewModel.kt]: nit: remove space between `()` and `:`

Comment (sagar-viradiya) [reply app/src/main/java/com/google/samples/apps/sunflower/viewmodels/GardenPlantingListViewModel.kt]: Sure

**Comment by sagar-viradiya** [comment]
Done

**Pull request merged**